ajax/libs/d3kit/3.2.0/d3kit-es.min.js

```diff
@@ -0,0 +1 @@
+// Error: Unexpected token: punc ({)
```

Comment:
@PeterDaveHello Do you mean remove `d3kit-es.min.js`, which would only left d3kit-es.js?